wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -529,6 +531,22 @@ def _update_run_states(self) -> None:
                 _logger.debug(f"Failed to get queued_run.state: {e}")
                 rqi_state = None
 
+            if run.state == RunState.PREEMPTED:
```

Comment:
i dont think our runs will actually be in an pre-empted state unfortunately. Not sure how we can get into this state, maybe legacy sweeps sets this?